src/manifest_parser_test.cc

```diff
@@ -24,7 +24,7 @@
 struct ParserTest : public testing::Test,
                     public ManifestParser::FileReader {
   void AssertParse(const char* input) {
-    ManifestParser parser(&state, this);
+    ManifestParser parser(&state, this, false);
```

Comment:
I probably should've made it an enum parameter, not a bool parameter. If you want, you can either add all the comments or change it to enum; or you can do nothing and I'll do the enum thing eventually.